docs/onboarding.rst

```diff
@@ -68,53 +68,64 @@ passphrase before continuing with the next section.
 -----------------------------------------------------
 
-Since the *Journalist Interface* is an ATHS, we need to set up the
-*Journalist Workstation* to auto-configure Tor just as we did with the
-*Admin Workstation*. The procedure is essentially identical, except the
+Since the *Journalist Interface* is an authenticated onion service, you must
+set up the *Journalist Workstation* to auto-configure Tor, similarly to 
+the *Admin Workstation*. The procedure is essentially identical, except the
 SSH configuration will be skipped, since only admins need
 to access the servers over SSH.
 
-.. tip:: Copy the files ``app-journalist-aths`` and ``app-source-ths`` from
-         the *Admin Workstation* via the Transfer Device. Place these files
-         in ``~/Persistent/securedrop/install_files/ansible-base`` on the
-         *Journalist Workstation*, and the ``./securedrop-admin tailsconfig``
-         tool will automatically use them. Don't forget to securely delete
-         these files from the *Transfer Device* when you're done, by
-         right-clicking them in the file manager and selecting **Wipe**.
+- First, boot into the *Admin Workstation*. If your instance has not been set up
+  to use v3 onion services, copy the following v2 service files to a *Transfer Device*:
 
-.. warning:: Do **not** copy the files ``app-ssh-aths`` and ``mon-ssh-aths``
+  .. code-block:: none
+ 
+    ~/Persistent/securedrop/install_files/ansible_base/app-source-ths
+    ~/Persistent/securedrop/install_files/ansible_base/app-journalist-ths
+
+  If your instance was set up to use v3 services, copy the following files instead:
+
+  .. code-block:: none
+
+    ~/Persistent/securedrop/install_files/ansible_base/app-sourcev3-ths
+    ~/Persistent/securedrop/install_files/ansible_base/app-journalist.auth_private
+
+  Then, boot into the new *Journalist Workstation* USB.
+
+- Install the SecureDrop application code on the workstation's persistent volume,
+  following the documentation for :ref:`cloning the SecureDrop
+  repository <Download the SecureDrop repository>`.
+
+- Copy the files from the *Transfer Device* to ``~/Persistent/securedrop/install_files/ansible-base``
+
+- Open a terminal and run the following commands:
+
+  .. code:: sh
+ 
+    cd ~/Persistent/securedrop
+    ./securedrop-admin setup
+    ./securedrop-admin tailsconfig
+
+  .. note:: The ``setup`` command may take several minutes, and may fail partway
+            due to network issues. If so, run it again before proceeding.
+
+- Once the ``tailsconfig`` command is complete, verify that the *Source* and
+  *Journalist Interfaces* are accessible at their v2 addresses via the 
+  SecureDrop desktop shortcuts.
+
+- Securely wipe the files on the *Transfer Device*, by right-clicking them
+  in the file manager and selecting **Wipe**.
+
+.. warning:: Do **not** copy the ``app-ssh-aths``, ``mon-ssh-aths``,
+             ``app-ssh.auth_private``, or ``mon-ssh.auth_private`` files
```

Comment:
suggestion: here we could also say to not copy `tor_v3_keys.json `to journalist workstations since this file also contain administrator secrets for SSH

Reply:
:white_check_mark: